tensorflow/lite/g3doc/inference_with_metadata/task_library/bert_nl_classifier.md

```diff
@@ -120,6 +120,38 @@ See the
 [source code](https://github.com/tensorflow/tflite-support/blob/master/tensorflow_lite_support/cc/task/text/bert_nl_classifier.h)
 for more details.
 
+## Run inference in Python
+
+### Step 1: Install the pip package
+
+```
+pip install tflite-support
+```
+
+### Step 2: Using the model
+
+```python
+# Imports
+from tflite_support.task import text
+from tflite_support.task import core
+from tflite_support.task import processor
+
+# Initialization
+base_options = core.BaseOptions(file_name=model_path)
+options = text.BertNLClassifierOptions(base_options=base_options)
```

Comment:
As the default initializer is the only initializer supported all three NLP tasks, let's show only it.

```
classifier = text.BertNLClassifier.create_from_file(model_path)
```

Reply:
@kinaryml

Reply:
fixed